sdk/communication/azure-communication-jobrouter/azure/communication/jobrouter/_router_administration_client.py

```diff
@@ -0,0 +1,1422 @@
+            exception_rules: Optional[Dict[str, ExceptionRule]],
+            name: Optional[str],
+            **kwargs: Any
+    ) -> ExceptionPolicy:
+        """ Update an exception policy.
+
+        :param str exception_policy_id: Id of the exception policy.
+
+        :keyword exception_rules: (Optional) A dictionary collection of exception rules on the exception
+          policy. Key is the Id of each exception rule.
+        :paramtype exception_rules: Optional[Dict[str, ~azure.communication.jobrouter.ExceptionRule]]
+
+        :keyword Optional[str] name: The name of this policy.
+
+        :return: ExceptionPolicy
+        :rtype: ~azure.communication.jobrouter.ExceptionPolicy
+        :raises: ~azure.core.exceptions.HttpResponseError, ValueError
+        """
+
+    @distributed_trace
+    def update_exception_policy(
+            self,
+            exception_policy_id: str,
+            exception_policy: ExceptionPolicy = ExceptionPolicy(),
+            **kwargs: Any
+    ) -> ExceptionPolicy:
+        """ Update an exception policy.
+
+        :param str exception_policy_id: Id of the exception policy.
+
+        :param exception_policy: An instance of exception policy. This is a positional-only parameter.
+          Please provide either this or individual keyword parameters.
+        :type exception_policy: ~azure.communication.jobrouter.ExceptionPolicy
+
+        :keyword exception_rules: (Optional) A dictionary collection of exception rules on the exception
+          policy. Key is the Id of each exception rule.
+        :paramtype exception_rules: Optional[Dict[str, ~azure.communication.jobrouter.ExceptionRule]]
+
+        :keyword Optional[str] name: The name of this policy.
+
+        :return: ExceptionPolicy
+        :rtype: ~azure.communication.jobrouter.ExceptionPolicy
+        :raises: ~azure.core.exceptions.HttpResponseError, ValueError
+
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/exception_policy_crud_ops.py
+                :start-after: [START update_exception_policy]
+                :end-before: [END update_exception_policy]
+                :language: python
+                :dedent: 8
+                :caption: Using a RouterAdministrationClient to update an exception policy
+        """
+        if not exception_policy_id:
+            raise ValueError("exception_policy_id cannot be None.")
+
+        patch = ExceptionPolicy(
+            name = kwargs.pop('name', exception_policy.name),
+            exception_rules = kwargs.pop('exception_rules', exception_policy.exception_rules)
+        )
+
+        return self._client.job_router_administration.upsert_exception_policy(
+            id = exception_policy_id,
+            patch = patch,
+            **kwargs
+        )
+
+    @distributed_trace
+    def get_exception_policy(
+            self,
+            exception_policy_id: str,
+            **kwargs: Any
+    ):
+        #  type: (...) -> ExceptionPolicy
```

Comment:
Move this comment into the method signature return type on line 325.

Reply:
Have changed it